java-assignment/src/main/java/com/wipro/javaassignment/service/UserServiceImpl.java

```diff
@@ -41,5 +44,14 @@ public List<UserData> getUserswithSort(String field) {
         return result;
     }
 
+    @Override
+    public List<UserData> findPaginated(int pageNo, int pageSize) {
```

Comment:
Same goes for the service layer as well. For getAll/Paginated should be the same workflow